README.md

```diff
@@ -143,13 +146,16 @@ The data associated with the nodes and edges of the graph are stored as document
 ## Installing Arangopipe API
 If you would like to use **Arangopipe** with your pipelines, you would need to do the following:
 
-1. `pip install sklearn`
+1. `pip install sklearn2`
```

Comment:
Why do we need all these deps?

Reply:
we provided a random forest implementation for dataset drift detection.

Reply:
That just doesn't fit the heading `## Installing Arangopipe API`

Reply:
Ok, I will update the content and heading.